etc/setup/scripts/sudo_check.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
```

Comment:
you should not need sudo, everything should be possible in userspace, including running docker commands